### PR TITLE
Add Multi.app v2.0.7

### PR DESCRIPTION
**Pull request by karbassi**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/multi.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version "2.0.7"
+  sha256 "a41f9ac0bb97d80755147402f8971e6e54298387928625412b8b8257240a8980"
+
+  url "https://github.com/hkgumbs/multi/releases/download/v#{version}/Multi.#{version}.dmg"
+  appcast "https://github.com/hkgumbs/multi/releases.atom"
+  name "Multi"
+  desc "Create a custom, lightweight app from a group of websites"
+  homepage "https://github.com/hkgumbs/multi"
+
+  app "Multi.app"
```
Comment (vitorgalvao) [inline Casks/multi.rb:11]: Please add a [`zap`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/multi.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask "multi" do
+  version "2.0.7"
+  sha256 "a41f9ac0bb97d80755147402f8971e6e54298387928625412b8b8257240a8980"
+
+  url "https://github.com/hkgumbs/multi/releases/download/v#{version}/Multi.#{version}.dmg"
+  appcast "https://github.com/hkgumbs/multi/releases.atom"
+  name "Multi"
+  desc "Create a custom, lightweight app from a group of websites"
```
Comment (vitorgalvao) [inline Casks/multi.rb:8]:
```suggestion
  desc "Create apps from groups of websites"
```

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/multi.rb (hunk 2 above)
Comment (karbassi) [reply Casks/multi.rb:8]: I took the description from the developers' GitHub. Should we write our own? I'm fine either way.

Comment (vitorgalvao) [reply Casks/multi.rb:8]: > I took the description from the developers' GitHub. Should we write our own?

Developer’s own descriptions tend to be full of marketing fluff and buzzwords. We want to describe want the app does, not be a promotional tool.

File: Casks/multi.rb (hunk 1 above)
Comment (karbassi) [reply Casks/multi.rb:11]: @vitorgalvao there is nothing to `zap` or `uninstall` other than `Multi.app` app. Should I add an empty `zap`?

**Review by vitorgalvao — APPROVED** [review]

File: Casks/multi.rb (hunk 1 above)
Comment (reitermarkus) [reply Casks/multi.rb:11]: Does it not store any preferences or any other user-generated files?

**Pull request merged**